File: lib/py/uapi/internal/binary/DecodeBody.py
from typing import TYPE_CHECKING, cast

if TYPE_CHECKING:
    from uapi.internal.binary.BinaryEncoding import BinaryEncoding


def decode_body(encoded_message_body: dict[object, object], binary_encoder: 'BinaryEncoding') -> dict[str, object]:
    from uapi.internal.binary.DecodeKeys import decode_keys

    return cast(dict[str, object], decode_keys(encoded_message_body, binary_encoder))

File: lib/ts/uapi/internal/binary/DecodeBody.ts
import { BinaryEncoding } from 'uapi/internal/binary/BinaryEncoding';
import { decodeKeys } from 'uapi/internal/binary/DecodeKeys';

export function decodeBody(encodedMessageBody: Map<any, any>, binaryEncoder: BinaryEncoding): Record<string, any> {
    return decodeKeys(encodedMessageBody, binaryEncoder);
}
